units/en/unitbonus5/introduction.mdx

```diff
@@ -0,0 +1,21 @@
+# Introduction:
+
```

Comment:
<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit13/thumbnail.png" alt="Unit bonus 4 thumbnail"/>

Reply:
Code is below